src/main/scala/org/globalforestwatch/layers/DeadwoodCarbonEmisYear.scala

```diff
@@ -5,6 +5,7 @@ import org.globalforestwatch.grids.GridTile
 case class DeadwoodCarbonEmisYear(gridTile: GridTile, model: String="standard")
   extends FloatLayer
     with OptionalFLayer {
-  val model_suffix = if (model == "standard") "" else s"__$model"
-  val uri: String = s"$basePath/gfw_deadwood_carbon_stock_in_emissions_year$model_suffix/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg/geotiff/${gridTile.tileId}.tif"
+  //    val model_suffix = if (model == "standard") s"__standard" else s"__$model"
+  val model_suffix = s"__$model"
```

Comment:
Codacy found an issue: [Field name 'model_suffix' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)